webapp/channels/src/components/custom_status/custom_status_suggestion.tsx

```diff
@@ -42,25 +41,19 @@ const CustomStatusSuggestion: React.FC<Props> = (props: Props) => {
     };
 
     const clearButton = handleClear ? (
-        <div
-            className='suggestion-clear'
-        >
-            <OverlayTrigger
+        <div className='suggestion-clear'>
+            <WithTooltip
                 delayShow={Constants.OVERLAY_TIME_DELAY}
                 placement='top'
-                overlay={
-                    <Tooltip id='clear-recent-custom-status'>
-                        {'Clear'}
-                    </Tooltip>
-                }
+                tooltipText={'Clear'}
```

Comment:
Out of the scope, but can we translate this string? instantiate `intl` by doing `const intl = useIntl();` and then here something like:

```suggestion
                tooltipText={intl.formateMessage({id: 'input.clear', defaultMessage: 'Clear'})}
```